pydis_site/apps/content/resources/guides/python-guides/setting-different-statuses-on-your-bot.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Setting Different Statuses to Set Your Bot
+description: How to personalize your Discord bot status
+---
+**Please note:**
+
+If you want to change the bot status, it is suggested to not do it during the on_ready event, since it would be called
+many times and making an API call on that event has a chance to disconnect the bot.
+Instead, set the desired status using the  activity / status kwarg of commands.Bot, for example
+`bot = commands.Bot(command_prefix="!", activity=..., status=...)`
```

Comment:
Can you make this a codeblock with formatting and maybe make it clear that the user should keep reading to understand what to put for the kwargs? That would make a nice gateway into the next part.